src/clients/storage_auto.rb

```diff
@@ -60,8 +63,9 @@ def main
         @ret = []
       # Change configuration (run AutoSequence)
       elsif @func == "Change"
-        @ret = StorageDialog()
-        UI.CloseDialog
+        storage_dialog = build_storage_dialog
+        @ret = storage_dialog.run
+        AutoinstPartPlan.Import(storage_dialog.partitioning.to_hashes) if @ret == :next
```

Comment:
when you touch it, I think it would be nice to use `::Installation::AutoClient` and also have some tests for it.

Reply:
also comment *why* it needs to import changes would be nice

Reply:
Yes, I thought about that but the PR is already too big, so I would rather prefer to do it in a separate PR. But, defintively, I should do it.